nixos/modules/config/console.nix

```
@@ -73,8 +73,8 @@ in
     };
 
     colors = mkOption {
-      type = types.listOf types.str;
-      default = [];
+      type = with types; listOf (strMatching "[a-zA-Z0-9]{6}");
```

```suggestion
      type = with types; listOf (strMatching "[[:xdigit:]]{6}");
```
This should match exactly an hex RGB colour.

---

:+1: